Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '^':
+        print('{} ^ {} = '.format(number_1, number_2), end = '')
+        return number_1 ** number_2
```

Comment:
Just a suggestion, can we have a switch operator instead of if else statements, as adding cases would be more readable than keeping adding if else statements?

Reply:
> Just a suggestion, can we have a switch operator instead of if else statements, as adding cases would be more readable than keeping adding if else statements?

Python doesn't have switch case statement.